packages/angular_devkit/architect/node/node-modules-architect-host.ts

```diff
@@ -100,16 +100,26 @@ export class WorkspaceNodeModulesArchitectHost implements ArchitectHost<NodeModu
     if (targetSpec === undefined) {
       return null;
     }
-    if (
-      target.configuration
-      && !(targetSpec['configurations'] && targetSpec['configurations'][target.configuration])
-    ) {
-      throw new Error(`Configuration '${target.configuration}' is not set in the workspace.`);
+
+    let additionalOptions = {};
+
+    if (target.configuration) {
+      const configurations = target.configuration.split(',');
```

Comment:
I think we should trim the elements here in case some does something like `-c "one, two, three"`

Reply:
That's a good idea. Added.